lms/templates/seq_module.html

```diff
@@ -62,16 +62,20 @@
             id="tab_${idx}">
             <span class="icon fa seq_${item['type']}" aria-hidden="true"></span>
             % if 'complete' in item:
-              <span
-                class="fa fa-check-circle check-circle ${"is-hidden" if not item['complete'] else ""}"
-                style="color:green"
-                aria-hidden="true"
-              ></span>
+              <span class="check-circle ${"is-hidden" if not item['complete'] else ""}">
```

Comment:
The `check-circle` and `bookmark-icon` classes are used by selectors in [this display JS file](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/sequence/display.js#L415)